### PR TITLE
Remove-portal

### PR DESCRIPTION
**Pull request by freignat91**
Portal has been moved to repository: https://github.com/appcelerator/portal

Remove portal for amp repository

These vendor was used by portal to check if they can be removed:
- github.com/gorilla/mux
- github.com/gorilla/context
- github.com/phyber/negroni-gzip
- github.com/urfave/negroni

### CODE REVIEW

File: platform/stacks/portal.stack.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ networks:
 services:
 
   portal:
-    image: appcelerator/portal:${TAG:-0.11.0}
+    image: appcelerator/portal:latest
```
Comment (bquenin) [inline platform/stacks/portal.stack.yml]: Shouldn't we move the stack file in the portal project as well?

**Review by bquenin — COMMENTED** [review]
LGTM, just one comment

File: platform/stacks/portal.stack.yml (hunk 1 above)
Comment (freignat91) [reply platform/stacks/portal.stack.yml]: I don't think so, we need a way to launch the portal service for amp cluster command

Comment (freignat91) [reply platform/stacks/portal.stack.yml]: but we can replace 'latest' by the current amp version

Comment (bquenin) [reply platform/stacks/portal.stack.yml]: ok, then yes, maybe keep the amp tagged version

Comment (freignat91) [reply platform/stacks/portal.stack.yml]: ok, i gonna change it

**Review by bquenin — APPROVED** [review]
LGTM

**Pull request merged**